src/flash.js

```diff
@@ -182,12 +195,22 @@ class Flash extends FakeEventTarget implements IEngine {
   constructor(source: PKMediaSourceObject, config: Object) {
     super();
     this._el = Utils.Dom.createElement('div');
-    this._init(source, config);
+    this.init(source, config);
   }
 
-  attachMediaSource(): void {}
+  attachMediaSource(): void {
+    this.init(this._source, this._config);
+    this._startTimeAttach = this._lastTimeDetach;
```

Comment:
`this. _startTimeAttach` is unnecessary. just use `this._lastTimeDetach`
see my comments https://github.com/kaltura/playkit-js/pull/425